src/content/docs/agents/net-agent/attributes/enable-disable-attributes-net.mdx

```diff
@@ -211,32 +211,33 @@ New Relic follows these rules to determine which attributes to include or exclud
       <include>foo</include>
     </attributes>
     <transactionEvents enabled="true">
-      <attributes enabled=”true”>
+      <attributes enabled="true">
         <exclude>foo</exclude>
+      </attributes>
     </transactionEvents>
     ```
 
     Example output:
 
     ```
     Keys passed in: foo
-    Keys included for transaction events: 
+    Keys included for transaction events:
     Keys included for other destinations: foo
     Keys excluded for transaction events: foo
     ```
   </Collapser>
 </CollapserGroup>
 
-## Deprecated properties [#deprecated]
+## Obsolete properties [#obsolete]
```

Comment:
Not sure if other docs redirect to this anchor. I would just change the title like you did but keep the anchor as  original: `## Obsolete properties [#deprecated]`

Reply:
@barbnewrelic, great catch! I switched this anchor back!